gapir/cc/replay_service.h

```diff
@@ -89,6 +89,8 @@ class ReplayService {
     Payload(std::unique_ptr<replay_service::Payload> protoPayload);
 
     ~Payload();
+
+    Payload() = delete;
```

Comment:
Is this part of the change?

Reply:
It's an oversight I noticed while looking at where the Payload comes from. I thought I'd correct it when I saw it.